kalite/distributed/settings.py

```diff
@@ -37,6 +36,7 @@ def USER_FACING_PORT():
 # TODO(bcipolli): change these to "login" and "logout", respectively, if/when
 #  we migrate to a newer version of Django.  Older versions require these
 #  to be set if using the login_required decorator.
+# TODO(benjaoming): Use reverse_lazy for this sort of stuff
 LOGIN_URL = "/securesync/login/"
 LOGOUT_URL = "/securesync/logout/"
```

Comment:
When you use for instance the `@login_required` decorator, you'll be redirected to settings.LOGIN_URL when the user isn't logged in.. this is the default URLs that django.contrib.auth uses except in the django admin.